docs/ui/emacs.rst

```diff
@@ -159,9 +161,12 @@ is open.
 
 You can toggle the interactive mode with ``C-c C-i``. If you were to do
 so, you still can navigate the proofs with ``C-c C-p`` and ``C-c C-n``,
-and show the goals at the current line with ``C-C C-c``. As always with
-emacs, if you were to be dissatisfied with these keybindings, you can
-change them easily!
+jump between proofs with ``C-c C-b`` and ``C-c C-f``, and show the goals
+at the current line with ``C-c C-c``. As always with emacs, if you were
+to be dissatisfied with these keybindings, you can change them easily!
+
+Clicking on a goal in the ``*Goals*`` buffer focuses it in the proof script
+by inserting a ``focus G`` tactic.
```

Comment:
I suggest: Clicking on the i-th goal of the ``*Goals*`` buffer puts the focus on it by inserting a ``focus i`` tactic in the proof script.